### ISSUE TITLE
Call Pet pop-out bugs when switching specs

### ISSUE THREAD

**Issue by OrbitStorm** [failcoder/Supervillain_UI#141]
When switching spec as a Hunter, the Call Pet pop-out will frequently bug and pop the menu open in a totally inconvenient manner.

For example: I have my Call Pet icon placed in the sidebar on the right side of my screen (Action Bar 4) and when switching specs, instead of the menu opening horizontally, it will open vertically thereby making it virtually impossible to select a pet without doing a /reload to correct the problem.

The same thing happens if you move Call Pet to one of the bottom action bars -- it'll just pop open horizontally when you don't want it to (immediately after switching specs).

In short, having to /reload every time I switch specs as a Hunter is quite tiresome. :(

**Comment by OrbitStorm** [comment]
Just to add, this seems to be a problem for Warlocks as well with Summon Demon and it doesn't appear to be limited to spec changes either (death will cause the same issue).
